src/redux/actions/experiments/createExperiment.js

```diff
@@ -7,52 +7,43 @@ import {
   EXPERIMENTS_ERROR,
   EXPERIMENTS_SAVING,
 } from 'redux/actionTypes/experiments';
-import { experimentTemplate } from 'redux/reducers/experiments/initialState';
 
 import endUserMessages from 'utils/endUserMessages';
 
 import handleError from 'utils/http/handleError';
 
 const createExperiment = (
-  newExperimentName,
+  name, description,
 ) => async (dispatch) => {
   const createdDate = moment().toISOString();
-
   const experimentId = hash.MD5(createdDate);
 
-  const newExperiment = {
-    ...experimentTemplate,
+  const newExperimentProperties = {
     id: experimentId,
-    name: newExperimentName,
-    projectUuid: experimentId,
-    createdDate,
+    name,
+    description,
   };
 
-  const { id, name, description } = newExperiment;
-  const experimentToSend = { id, name, description };
-
-  const url = `/v2/experiments/${experimentId}`;
-
   dispatch({
     type: EXPERIMENTS_SAVING,
   });
 
   try {
     await fetchAPI(
-      url,
+      `/v2/experiments/${experimentId}`,
       {
         method: 'POST',
         headers: {
           'Content-Type': 'application/json',
         },
-        body: JSON.stringify(experimentToSend),
+        body: JSON.stringify(newExperimentProperties),
       },
     );
 
     dispatch({
       type: EXPERIMENTS_CREATED,
       payload: {
-        experiment: newExperiment,
+        experiment: { createdDate, ...newExperimentProperties },
```

Comment:
remember to take a look at using the createdDate from the api as part of another set of PRs